File: UtilitiesVB/Solution Interfaces/IAppAutoFileObjects.vb
Imports UtilitiesCS

Public Interface IAppAutoFileObjects
    Property Conversation_Weight As Long
    Property LngConvCtPwr As Long
    Property MaxRecents As Long
    Property RecentsList As IRecentsList(Of String)
    Property CTFList As CtfIncidenceList
    Property CommonWords As ISerializableList(Of String)
    Property SuggestionFilesLoaded As Boolean
    Property SmithWatterman_MatchScore As Integer
    Property SmithWatterman_MismatchScore As Integer
    Property SmithWatterman_GapPenalty As Integer
End Interface

File: UtilitiesVB/Solution Interfaces/IAppAutoFileObjects.cs
using UtilitiesCS;
using UtilitiesCS.ReusableTypeClasses;

namespace UtilitiesVB
{

    public interface IAppAutoFileObjects
    {
        int Conversation_Weight { get; set; }
        int LngConvCtPwr { get; set; }
        int MaxRecents { get; set; }
        IRecentsList<string> RecentsList { get; set; }
        CtfIncidenceList CTFList { get; set; }
        ISerializableList<string> CommonWords { get; set; }
        bool SuggestionFilesLoaded { get; set; }
        int SmithWatterman_MatchScore { get; set; }
        int SmithWatterman_MismatchScore { get; set; }
        int SmithWatterman_GapPenalty { get; set; }
        public ISubjectMapSL SubjectMap { get;  }
        ISubjectMapEncoder Encoder { get; }
    }
}